core/jvm/src/main/scala/scalaz/zio/ManagedApp.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package scalaz.zio
+
+trait ManagedApp extends DefaultRuntime {
```

Comment:
Thank you for working on this! Would it be possible to reuse the existing App so we won’t have to duplicate the implementation?

Reply:
Yep, make sense. Thank you.

Reply:
Yeah you don't have to override it. You can keep an instance of `App` as a member in this trait, and delegate to its main method from our main method.

Reply:
Yeah, you are right. Thanks!